src/core/expr/fexpr_nth.cc

```diff
@@ -64,19 +63,15 @@ class FExpr_Nth : public FExpr_Func {
         case SType::INT64:   return make<int64_t>(std::move(col), n, gby);
         case SType::FLOAT32: return make<float>(std::move(col), n, gby);
         case SType::FLOAT64: return make<double>(std::move(col), n, gby);
-        //case SType::STR32:   return make<CString>(std::move(col), n, gby);
+        //case SType::STR32:   return make<CString>(std::move(col), n, gby); -- any ideas on how to make this work @oleksiyskononenko
```

Comment:
What exactly doesn't work here?

Reply:
the `make` function throws an error for string functions, and I can't figure out the error for that